examples/research_projects/stack_llama_2/scripts/dpo_llama2.py

```diff
@@ -74,6 +75,11 @@ class ScriptArguments:
         },
     )
 
+    device_map_local_process_idx: Optional[bool] = field(
+        default=True, metadata={"help": "whether to device map model to local process index, see "
+                                "https://github.com/huggingface/trl/issues/1348"}
+    )
```

Comment:
```suggestion
```
I think we can remove that and always create `device_map = "device_map": {"": Accelerator().local_process_index}`

Can you just make `gradient_checkpointing_kwargs` configurable here? 🙏

Reply:
updated.